scss/_patterns_suru.scss

```
@@ -0,0 +1,20 @@
+@import 'settings';
+
+@mixin vf-p-suru {
+  .p-suru {
+    aspect-ratio: 5.1775700934579; // exact aspect ration of the image
+    background-image: url('#{$assets-path}7f34ade9-website_suru_25.jpg');
+    background-size: contain;
+    display: none; // only show suru on large screens
+    margin: 0 auto;
+    max-width: $grid-max-width;
+
+    @media (min-width: $breakpoint-large) {
+      display: block;
+    }
```

```suggestion
  .p-suru {
      display: none;

    @media (min-width: $breakpoint-large) {
      display: block;
      aspect-ratio: 5.1775700934579; // exact aspect ration of the image
      background-image: url('#{$assets-path}7f34ade9-website_suru_25.jpg');
      background-size: contain;
      margin: 0 auto;
      max-width: $grid-max-width;
    }
```

I would personally move all the display rules in the media query like so
But idk if it's a pattern we use elsewhere

---

I find it a bit cleaner this way, and in the end I don't think it really matters too much.